.travis.yml

```diff
@@ -1,9 +1,18 @@
 language: php
-
 php:
-    - 5.3
     - 5.4
```

Comment:
Hm, what about removing the support for such old php versions? I think the performance boost we could have by using PHP 7 could be the last thing needed for this lib to become Mr. worldwide.

Also, if we would support only > PHP 5.6 

we could do `$addition = new AdditionOperator(SimpleNumber::class);` instead of `$addition = new AdditionOperator('SimplePHPEasyPlus\Number\SimpleNumber');`

Reply:
That's not my decision, as I'm not a member of this project, only a contributor.

When I downloaded the project and tried to execute it with Travis CI, I couldn't because the Ubuntu's Trusty version doesn't support the PHP's 5.3 release. So keeping the ability to perform all the previous tests and support those for new PHP versions, I ended up reconfiguring the Travis CI configuration file.